spk/firebird/src/service-setup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+FBGUARD_BIN_FILE="${SYNOPKG_PKGDEST}/bin/fbguard"
+
+# We set onetime because we want the systemd to take care about the failures
+SERVICE_COMMAND="${FBGUARD_BIN_FILE} -pidfile ${PID_FILE} -daemon -onetime"
+
+SYSDBA_PASSWORD_FILE="${SYNOPKG_PKGDEST}/SYSDBA.password"
+
+service_postinst () {
+    set -x
+    cd "$SYNOPKG_PKGDEST" || exit 1
+    bin/gbak -rep msg.gbak msg.fdb
+    bin/build_file
+
+    # Make the message file accessible so other users can use isql
+    chmod go+r firebird.msg
```

Comment:
Did you test this on DSM 7?
I suppose this will fail when not run as root?

Reply:
Hmm, it worked for me on 7.1. I will observe it more closely to confirm.

Reply:
It works for me on 7.1